client/data/marketplace/use-es-query.ts

```diff
@@ -134,7 +138,7 @@ export const getESPluginsInfiniteQueryParams = (
 			pageSize,
 			locale: getWpLocaleBySlug( ( options.locale || locale ) as LanguageSlug ),
 		} );
-	return [ cacheKey, fetchFn ];
+	return { queryKey, queryFn };
```

Comment:
It's worth noting that `getESPluginsInfiniteQueryParams()` is also used here:

https://github.com/Automattic/wp-calypso/blob/8f1fa912356c59738ed483a536f84bb85f191ca1/client/my-sites/plugins/controller-logged-out.js#L46

and here:

https://github.com/Automattic/wp-calypso/blob/8f1fa912356c59738ed483a536f84bb85f191ca1/client/my-sites/plugins/controller-logged-out.js#L55

and we might need to do some updates to the functions there to accommodate that change.

Reply:
Great catch @tyxla! I totally missed those (I thought I checked references). I'll make sure to double-check and add adjustments accordingly 👍🏻

Reply:
https://github.com/Automattic/wp-calypso/pull/77658/commits/2bc177f3509195f4f755d4dc1dc179de4dd3f5bf should cover this and all other mentioned cases.

Reply:
Nice 👍